src/components/Button.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Text, View, StyleSheet, TouchableOpacity } from 'react-native';
+
+export type ButtonProps = {
+    title: string;
+    btnColor?: string;
+    onPress?: () => void;
+}
+
+const styles = StyleSheet.create({
+  button: {
+    height: 50,
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderRadius: 8,
+    backgroundColor: '#000',
+    borderWidth: 2,
+    borderColor: '#000',
+    marginTop: 20,
+    marginLeft: 15,
+    marginRight: 15,
+  },
+  buttonLight: {
+    backgroundColor: '#fff',
+    borderWidth: 2,
+    borderColor: '#000'
+  },
+  text: {
+    color: '#000',
+    fontSize: 16
+  },
+  textLight: {
+    color: '#fff',
+    fontSize: 16
+  }
+});
+
+
+export default function Button(props: ButtonProps) {
+  const { title, btnColor, onPress } = props;
```

Comment:
The second line is redundant. You can destructure props directly in first line and it'll improve readability, like:
```
export default function Button({ title, btnColor, onPress }: ButtonProps) {
```

Also, `color` instead of `btnColor` should be enough, because it's already `Button` component, so developer won't get confused.